index.js

```diff
@@ -17,29 +18,27 @@ LESSCompiler.prototype.type = 'stylesheet';
 LESSCompiler.prototype.extension = 'less';
 
 LESSCompiler.prototype.compile = function(params, callback) {
-  var data = params.data;
   var path = params.path;
-
-  var parser = new less.Parser({
-    paths: [this.rootPath, sysPath.dirname(path)],
-    filename: path,
-    dumpLineNumbers: !this.optimize && this.config.dumpLineNumbers
-  });
-
-  parser.parse(data, function(error, tree) {
-    if (error != null) return callback(error.message);
-    var result, err;
-    try {
-      result = tree.toCSS();
-    } catch (ex) {
-      err = '' + ex.type + 'Error:' + ex.message;
-      if (ex.filename) {
-        err += ' in "' + ex.filename + ':' + ex.line + ':' + ex.column + '"';
-      }
-    }
-
-    return callback(err, {data: result});
-  });
+  
+  var filePath = sysPath.join(this.rootPath, path);
+  var fileContent = fs.readFileSync(filePath, { encoding: 'utf8' });
```

Comment:
Why are you re-reading the file when brunch provides the content already in `params.data`? 

In addition, the `Sync` form of `fs` methods should not be used.